LICENSE.txt

```diff
@@ -1,27 +1,13 @@
-Copyright (c) 2015-2018, conda-forge
```

Comment:
Please do not modify files outside of the recipes/ dir in a recipe PR,

Reply:
sorry, not sure how it got changed.